pkg/apis/appmesh/v1beta1/types.go

```diff
@@ -352,13 +352,17 @@ type VirtualNodeSpec struct {
 	// +optional
 	Backends []Backend `json:"backends,omitempty"`
 	// +optional
+	BackendDefaults *BackendDefaults `json:"backendDefaults,omitempty"`
+	// +optional
 	Logging *Logging `json:"logging,omitempty"`
 }
 
 type Listener struct {
 	PortMapping PortMapping `json:"portMapping"`
 	// +optional
 	HealthCheck *HealthCheckPolicy `json:"healthCheck,omitempty"`
+	// +optional
+	Tls *ListenerTls `json:"tls,omitempty"`
```

Comment:
nit: It's better to use `TLS` as field name to follow k8's convention Ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md
```All letters in the acronym should have the same case, using the appropriate case for the situation```
TLS in Ingress spec is using `TLS`

Reply:
Ok. Are you suggesting the field name is `TLS`? Orr the yaml/json? Or both?

> TLS in Ingress spec is using TLS

What field is this referring to?

Reply:
Clarification: Ingress appears to actually have this as all lowercase: `tls`

https://kubernetes.io/docs/concepts/services-networking/ingress/#tls

Reply:
@bcelenza FYI: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/networking/types.go#L264 🤣

Reply:
Haha, well consistently aligned with the formal definition I guess.

I think it makes sense to keep it lower case, because that's what we've done for dns: https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/deploy/all.yaml#L156

Reply:
@bcelenza the attribute is lower case. it's the field name in golang structure.

Reply:
Ah, thanks for the clarification.

Reply:
Ok, so just changing the field name in the struct to `TLS`, got it.